src/blog.rs

```diff
@@ -261,14 +270,22 @@ impl Blog {
         actions.push(FsAction::WriteFile {
             path: root.as_ref().join("atom.xml").into(),
             content: atom::Feed {
-                title: blog_title.clone(),
-                id: blog_title,
+                title: "BLOG TITLE TODO".into(),
```

Comment:
Why can't this remain being `blog_title`?

Reply:
good catch ty